docker.local/config/conductor.blobber-5.yaml

```diff
@@ -6,11 +6,45 @@ enable:
 sets:
   - name: "Minority Cases"
     tests:
+      - "Rollback tokenomics"
       - "Allocation state should be same after rollback"
       - "Allocation state should be same after repair"
 
 # test cases
 tests:
+  - name: "Rollback tokenomics"
```

Comment:
The test look incomplete to me

Reply:
Where is the part where you sync the value before it changes? Also the rollback will not run since the number of blobbers carrying the allocation is more than DataShards.

Reply:
I am maintaining list of allocations objects in CS store so fetching it directly from there.